README.md

```diff
@@ -20,16 +20,16 @@ To retrieve and generate the data, run the command below in the Terminal:
 ``` sh
 conda env create --file requirements.yaml 
 conda activate space-env
-curl -sSL https://raw.githubusercontent.com/NSAPH-Projects/space/master/data/data.sh | bash -s NN|XGBOOST BINARY|CONT SEED PATH
+curl -sSL https://raw.githubusercontent.com/NSAPH-Projects/space/master/data/data.sh | bash -s [NN or XGBOOST] [BINARY or CONT] SEED PATH
```

Comment:
this will only remain in the release that corresponds to CLeaR, correct?

Reply:
yes, we'll have a normal api very soon